tests/scripts/openwrt

```diff
@@ -43,7 +43,7 @@ readonly CONTAINER_NAME
 
 do_prepare()
 {
-    docker create --name "${CONTAINER_NAME}" --rm -v openwrt-bin:/home/build/openwrt/bin -it openwrt/sdk
+    docker create --name "${CONTAINER_NAME}" --rm -v openwrt-bin:/home/build/openwrt/bin -it openwrt/sdk:22.03.3
```

Comment:
Is this change intentional?

Reply:
Yes, current openwrt CI is broken because the openwrt/sdk:latest has incompatible changes that otbr can't build it the docker image (e.g. no `sudo`, no `/home/build/openwrt` directory). It's probably more stable to stick with the latest "good" build to unblock the CI on the main branch (for example, https://github.com/openthread/ot-br-posix/pull/1930).

I will ask the commits being merged separately.